using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -953,7 +1104,39 @@ Before using this API, see [Identify unique visitors](https://docs.adobe.com/con
 This API gets a custom Analytics visitor identifier, which has been set previously using [setVisitorIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#setidentifier). Callback with NSError if an unexpected error occurs or the request times out.
 
 {% tabs %}
-{% tab title="iOS" %}
+
+{% tab title="iOS (AEP 3.x)" %}
+
+### getVisitorIdentifier
+
+#### Syntax
+
+```swift
+static func getVisitorIdentifier(completion: @escaping (String?, Error?) -> Void)
+```
+
+#### Example
+
+**Swift**
+
+```swift
+Analytics.getVisitorIdentifier { (visitorId, error) in
+   // check the visitorIdentifier value or handle error
+}
+```
+
+**Objective-C**
+
+```objectivec
+[AEPMobileAnalytics getVisitorIdentifier:^(NSString * _Nullable visitorIdentifier, NSError * _Nullable error) {
+    // check the visitorIdentifier value or handle error
```

Comment:
let's reverse this `// Handle the error (if non-nil) or use the visitorIdentifier value`